### PR TITLE
publish to S3

### PR DESCRIPTION
**Pull request by anzecesar**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
-version := "0.3.9"
+version := "0.3.11"
 
-organization := "net.noerd"
+organization := "com.celtra"
 
 scalaVersion := "2.10.3"
 
+scalacOptions ++= Seq(
+    "-Xmax-classfile-name", "140"
```
Comment (gregakespret) [inline build.sbt:13]: @anzecesar Why do we need this? Some s3 restriction on the max filename length or something?

File: build.sbt [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
+)
+
 // Runtime Dependencies
 libraryDependencies ++= Seq(
     "commons-pool" % "commons-pool" % "1.5.5",
     "commons-dbcp" % "commons-dbcp" % "1.4",
     "commons-lang" % "commons-lang" % "2.6",
-    "joda-time" % "joda-time" % "2.1",
-    "org.joda" % "joda-convert" % "1.2"
+    "joda-time" % "joda-time" % "2.6",
```
Comment (gregakespret) [inline build.sbt:21]: Is the increasing of Joda version correlated to the publishing part? How?

Comment (anzecesar) [inline build.sbt:21]: It isn't

File: build.sbt (hunk 1 above)
Comment (anzecesar) [inline build.sbt:13]: No, this is not necessary. It's required to compile on ecryptfs, where I compiled it. It has no impact on anything else, so it stayed there

**Pull request merged**